src/js/components/projects/repositoryGitHubUsers.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// TODO:
+// - Make these useful cards.
+// - Insert these in the correct place in the document.
+// - The list of Repo users should probably be a modal, and a separate
+//   component, compared to the list of Project users, yes?
```

Comment:
Yep, I think that's right! As far as the markup...

- The cards should use the `Card` component from SLDS. See `components/orgs/cards.tsx` for an example.
- The list should be in a `Modal` (see `ConfirmDeleteModal` in `components/orgs/cards.tsx`), and use the `DataTable` component (see `components/commits/list.tsx`).